src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -312,6 +312,14 @@ def subvolumes_prefix
         children.size == 1 ? children.first.path : ""
       end
 
+      # Determines whether the snapshots (snapper) are activated
+      #
+      # @return [Boolean] true if snapshots are present
+      def snapshots?
```

Comment:
For already existing filesystems maybe this is our best chance to know whether the snapshots are configured. But for new filesystems we have `Filesystems::Btrfs.configure_snapper`

Reply:
`Filesystems::Btrfs.configure_snapper` can be checked if there is no `.snapshots` subvolume.